meta/applications/yc_s15.md

```diff
@@ -0,0 +1,402 @@
+
+**How long have the founders known one another and how did you meet? Have any
+of the founders not met in person?**
+
+We've known each other since September 2014 (6 months) and met after a mutual
+friend introduced Zach, who had just won PennApps, the University of
+Pennsylvania's hackathon, to Jonathan, who was a former lead organizer of
+PennApps.
+
+**How far along are you?**
+
+We ran a pilot last semester with 25 schools and reached 1000 students, 35% of
+whom are female (2x national average) and 18% of whom are minorities in tech
+(3x national average). At MHacks V and PennApps X, the two largest collegiate
+hackathons with >1000 attendees from schools including Stanford, MIT, and
+Harvard, half of the winning teams were led by high school students, all of
+whom are part of their local hackEDU chapter.
+
+**If you've already started working on it, how long have you been working and
+how many lines of code (if applicable) have you written?**
+
+We've been working on hackEDU full-time for 5 months.
+
+**Which of the following best describes your progress?**
+
+* [ ] Nothing Built
+* [ ] Sketches
+* [ ] Mocks / Renderings
+* [ ] Prototype
+* [x] Private Beta
+* [ ] Public Beta
+* [ ] Taking Preorders
+* [ ] Launched
+
+**Do you have revenue?**
+
+* [x] No
+* [ ] Yes
+
+**If you've applied previously with the same idea, how much progress have you
+made since the last time you applied? Anything change?**
+
+We ran our planned 25 school pilot, reaching 1,000 students, had Jonathan join
+as a co-founder, and brought on a full-time volunteer as well as two part-time
+volunteers.
+
+**If you have already participated or committed to participate in an incubator,
+"accelerator" or "pre-accelerator" program, please tell us about it.**
+
+We have not.
+
+**Why did you pick this idea to work on? Do you have domain expertise in this
+area? How do you know this is going to work?**
+
+During his first year of high school, Zach started a programming club because
+he wanted a community of other programmers at his school. By the end of the 2nd
+year, the students in his club launched games on the Android Play Store, won
+high-profile hackathons, founded a web design firm that landed real clients,
+and secured over $30,000 in investment. After meeting other high school students
+who had also started programming clubs, he decided to start hackEDU. Being 17,
+he is the age of our target demographic and has made himself a leader in the
+wider high school programming community. Zach runs HS Hackers, the largest
+online community for high school developers (Facebook shared a
+[post](https://www.facebookstories.com/stories/112524/building-a-community-for-high-schoolers-who-code)
+about it to 2.3m users).
+
+**What's new about what you plan to do?**
+
+* Instead of focusing on teaching a set curriculum at hack clubs, hackEDU
+  focuses on building all-inclusive families where members hack on things
+  together.
```

Comment:
Why is this important? What is the existing hack club model missing by not creating "families"?